src/main/scala/loamstream/googlecloud/CloudSdkDataProcClient.scala

```diff
@@ -105,10 +108,26 @@ object CloudSdkDataProcClient extends Loggable {
     gcloudTokens(config)("create")(tokens: _*)
   }
 
+  private[googlecloud] def toGoogleFormat(d: Duration): String = {
+    import scala.concurrent.duration._
+    
+    val coarsest = d.toCoarsest
+    
+    val timeUnitSuffix = coarsest.unit match {
+      case SECONDS => "s"
+      case MINUTES => "m"
+      case HOURS => "h"
+      case DAYS => "d"
+      case u => sys.error(s"Time unit ${u} not supported by Google; must be one of SECONDS, MINUTES, HOURS, or DAYS")
```

Comment:
Instead of error-ing out, can you instead just convert and use `d.toSeconds`?

Reply:
That's definitely an option.  The `to*` methods truncate values, like
```scala
scala> 500.milliseconds.toSeconds
res0: Long = 0
```
That's lame, but specifying the idle time for a *cluster* in milliseconds means effectively "zero seconds" anyway.  

Another option is to error out at config-file-parse time (effectively LS-start-time) instead of what this does, which is to error out at pipeline-execution-time.

But I can't see any harm in truncating some values, and that's marginally simpler than optionally failing at config-file-parse-time.  I'll do that.

Reply:
So the plot thickened: in addition to requiring a particular format for durations, Google enforces minimum and maximum values for the particular param in question.  People could just read the docs, but even so, I didn't want simple-to-fix typos or out-of-range errors to be delayed until the middle of a pipeline run.  

So I made it so this duration is specified directly in Google's format in `loamstream.conf`, and that LS checks that it's formatted properly and between min and max values at config-file-parse-time, which is effectively LS-start-time.